Source/WebCore/svg/graphics/filters/SVGFilter.h

```diff
@@ -49,6 +50,10 @@ class SVGFilter final : public Filter {
     
     FilterEffectVector effectsOfType(FilterFunction::Type) const final;
 
+    FilterResults* results() { return m_results.get(); }
+    WEBCORE_EXPORT FilterResults& ensureResults(const FilterResultsCreator& = [] { return makeUnique<FilterResults>(); });
```

Comment:
The implementation of this function is this:
```
FilterResults& SVGFilter::ensureResults(const FilterResultsCreator& resultsCreator)
{
    if (!m_results)
        m_results = resultsCreator();
    return *m_results;
}
```
If the default FilterResultsCreator is passed, this is going to be resolved to:
```
FilterResults& SVGFilter::ensureResults(const FilterResultsCreator& resultsCreator)
{
    if (!m_results)
        m_results = makeUnique<FilterResults>();
    return *m_results;
}
```
So this function should compile and it should return a reference to member `*m_results`.